charts/workspacemanager/templates/proxy-configmap.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: terra-workspace-manager-proxy-configmap
+  labels:
+    {{ if not .Values.devDeploy }}{{ template "workspacemanager.labels" }}{{ end }}
+data:
+  siteconf: |-
+    ServerAdmin ${SERVER_ADMIN}
+    ServerName {{ .Chart.Name }}.{{ .Release.Namespace }}.{{ .Values.certManager.dnsSuffix }}
```

Comment:
Ah yep, `dnsSuffix` should not be under `certManager` really. For existing Terra envs it will be set to `dsde-[env].broadinstitute.org` and instead of cert-manager creating certs for each app we will just sync the existing wildcard certs for the corresponding env from Vault. I can re-factor this real quick.